src/views/MoviesPage.vue

```diff
@@ -0,0 +1,130 @@
+<script>
+import { defineComponent } from 'vue';
+
+import BaseTextInput from '@/components/global/BaseTextInput.vue';
+import BaseSelect from '@/components/global/BaseSelect.vue';
+import MoviesContainer from '@/components/MoviesPage/MoviesContainer.vue';
+
+import { useMoviesStore } from '@/store/movies';
+const moviesStore = useMoviesStore();
```

Comment:
Same case as in `HomePage` file with `setup` 👌

Reply:
Done!